js/mailbridge.js

```diff
@@ -138,10 +139,15 @@ MailBridge.prototype = {
   __receiveMessage: function mb___receiveMessage(msg) {
     var implCmdName = '_cmd_' + msg.type;
     if (!(implCmdName in this)) {
-      this._LOG.badMessageType(msg.type);
+      logic(this, 'badMessageType', { type: msg.type });
       return;
     }
-    var rval = this._LOG.cmd(msg.type, this, this[implCmdName], msg);
+    logic(this, 'cmd', {
```

Comment:
The transformation of this "calls" log type changes behaviour, which is largely okay, but I think there might be some test fallout.  For context for other cases, the loggest calls mechanism:
* logs [name, ...args..., startTS, startSeqNo, endTS, endSeqNo, exception (or null), ...testOnlyArgs...]
* wraps the call in a try/catch and returns the return value of the function OR its exception if an exception was thrown.  In test mode, an exception being thrown is specially handled as an error by the _verify_ function in addition to the equivalence checks.

For this site:
* We don't need to log the function that we're invoking.  We should lose 'impl'
* The try/catch got lost.  I don't think this poses an issue at runtime because the exception propagation should only kill an onmessage handler, but it looks like we're potentially losing some important test coverage here.  While the "logic" logger should end up echoing the various console.error() stuff we have around into the log on disk, this won't cause a test to fail.

I think it does make sense to support a class of errors that are "this should absolutely never happen" and this should fail the tests without requiring the test to go out of its way to do it, and I think __receiveMessage throwing is an example of this.  One way to handle this might be to establish a convention of instrumenting such failure with a simple consistent type, perhaps like node.js's "uncaughtException" or "fatalError" and have GelamTest listen on this and have it trigger a rejection.  We could potentially also hook this up to "onerror" as well.  (Note that pre-this-PR loggest-style testing I don't think actually generates a failure for onerror, but ArbPL papers over this deficiency because the console.error is understood to have error semantics and so the ArbPL shows it in red.)

I don't have a strong opinion on how to do this, but think this is something we should at least put a hack-fix for this.  A follow-up bug for something more elegant is fine.